pdns/rcpgenerator.cc

```diff
@@ -643,7 +642,7 @@ void RecordTextReader::xfrText(string& val, bool multi, bool /* lenField */)
             char chr2 = d_string[d_pos + 1];
             char chr3 = d_string[d_pos + 2];
             if (chr2 >= '0' && chr2 <= '9' && chr3 >= '0' && chr3 <= '9') {
-              valid = true;
+              valid = 100 * (chr - '0') + 10 * (chr2 - '0') + chr3 - '0' < 256;
```

Comment:
```suggestion
              valid = (100 * (chr - '0') + 10 * (chr2 - '0') + chr3 - '0') < 256;
```

NFCI but my brain really likes it better this way